src/wp-admin/css/about.css

```diff
@@ -647,9 +650,10 @@
 @media screen and (max-width: 960px) {
 	.about__header-title h1 {
 		/* Fluid font size scales on browser size 600px - 960px. */
-		font-size: clamp(3rem, 13.33vw - 2rem, 6rem);
+		/*font-size: clamp(3rem, 13.33vw - 2rem, 6rem);*/
```

Comment:
If this is going to not be used, let's remove the block rather than comment out the property. Or was this accidental testing code you pushed?

Reply:
It's been removed. Thanks for catching that.